### DISCUSSION TITLE
Welcome to Firefiles Discussions!

### DISCUSSION THREAD

**Discussion by faisalsayed10** [faisalsayed10/firefiles#31] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by andrewn6** [comment]
hi there! my name is andrew and i write code

**Reply by faisalsayed10** [reply]
Hi andrew!!!
